docker-hub/access-tokens.md

```diff
@@ -28,12 +28,21 @@ Settings**.
 
 3. Select **Security > New Access Token**.
 
-      ![](images/hub-create-token.png)
-
 4. Add a description for your token. Use something that indicates where
 the token is going to be used, or set a purpose for the token.
 
-5. Copy the token that appears on the screen. Make sure you do this now:
+      ![](images/PAT-Menu.png)
```

Comment:
Looks like this image is replacing `hub-create-token.png`. Could you rename the new file to be the same name as the old name, and replace the existing image?

Reply:
Updated! Thank you!